doc/api/next_api_changes/removals.rst

```diff
@@ -71,7 +73,80 @@ Classes and methods
+
+- ``mlab.demean()`` (use ``mlab.detrend_mean()`` instead)
+- ``projections.process_projection_requirements()`` (no replacement)
+- ``path.get_paths_extents()``
+(use ``path.get_path_collection_extents()`` instead)
+
+- ``docstring.Appender`` (no replacement)
+- ``docstring.dedent()`` (use ``inspect.getdoc()`` instead)
+- ``docstring.copy_dedent()``
+  (use ``docstring.copy()`` and ``cbook.getdoc()`` instead)
+
+- ``ticker.OldScalarFormatter.pprint_val()`` (no replacement)
+- ``ticker.ScalarFormatter.pprint_val()`` (no replacement)
+- ``ticker.LogFormatter.pprint_val()`` (no replacement)
+- ``ticker.decade_down()`` (no replacement)
+- ``ticker.decade_up()`` (no replacement)
+
+- ``scale.LogTransformBase()`` (use ``scale.LogTransform()`` instead)
+- ``scale.InvertedLogTransformBase()`` (use ``scale.InvertedLogTransform()`` instead)
+- ``scale.Log10TransformBase()`` (use ``scale.LogTransform()`` instead)
+- ``scale.InvertedLog10TransformBase()`` (use ``scale.InvertedLogTransform()`` instead)
+- ``scale.Log2Transform()`` (use ``scale.LogTransform()`` instead)
+- ``scale.InvertedLog2Transform()`` (use ``scale.InvertedLogTransform()`` instead)
+- ``scale.NaturalLogTransform()`` (use ``scale.LogTransform()`` instead)
+- ``scale.InvertedNaturalLogTransform()`` (use ``scale.InvertedLogTransform()`` instead)
+
+- ``spines.Spine.is_frame_like()`` (no replacement)
+- ``text.is_math_text()`` (no replacement)
+- ``text.TextWithDash()`` (use ``text.Annotation`` instead)
+- ``textpath.TextPath.is_math_text()`` (no replacement)
+- ``textpath.TextPath.text_get_vertices_codes()``
+  (use ``textpath.text_to_path.get_text_path()`` instead)
+
+- ``widgets.SpanSelector.buttonDown`` property (no replacement)
+
+- ``mplot3d.proj3d.line2d()`` (no replacement)
+- ``mplot3d.proj3d.line2d_dist()`` (no replacement)
+- ``mplot3d.proj3d.line2d_seg_dist()`` (no replacement)
+- ``mplot3d.proj3d.mod()`` (use `numpy.linalg.norm`` instead)
+- ``mplot3d.proj3d.proj_transform_vec()`` (no replacement)
+- ``mplot3d.proj3d.proj_transform_vec_clip()`` (no replacement)
+- ``mplot3d.proj3d.vec_pad_ones()`` (no replacement)
+- ``mplot3d.proj3d.proj_trans_clip_points()`` (no replacement)
+
+- ``mplot3d.art3d.norm_angle()`` (no replacement)
+- ``mplot3d.art3d.norm_text_angle()`` (no replacement)
+- ``mplot3d.art3d.path_to_3d_segment()`` (no replacement)
+- ``mplot3d.art3d.paths_to_3d_segments()`` (no replacement)
+- ``mplot3d.art3d.path_to_3d_segment_with_codes()`` (no replacement)
+- ``mplot3d.art3d.paths_to_3d_segments_with_codes()`` (no replacement)
+- ``mplot3d.art3d.get_patch_verts()`` (no replacement)
+- ``mplot3d.art3d.get_colors()`` (no replacement)
+- ``mplot3d.art3d.zalpha()`` (no replacement)
+
+- ``mplot3d.axis3d.get_flip_min_max()`` (no replacement)
+- ``mplot3d.axis3d.Axis.get_tick_positions()`` (no replacement)
+
+- ``axisartist.axis_artist.UnimplementedException`` (no replacement)
+- ``axisartist.axislines.SimpleChainedObjects`` (no replacement)
```

Comment:
axis_grid1.mpl_axes.SimpleChainedObjects (sorry)

Reply:
That's not officially deprecated.

Reply:
I meant these are replacements (below too).